vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -162,3 +184,6 @@ async def create_speech(
             convert_span.end()
 
             return result
+        
+    def get_cache_key(self, text) -> str:
```

Comment:
this can be in `base_synthesizer` as well as additional method, no?

Reply:
yeah I thought about this, and maybe it can with Python Magic - but my first inclination was that in the base class sythesizer_config would not be the proper type to be able to call a subclassed synthesizer_config's `__hash__` function (since SynthesizerConfig itself doesnt have a __hash__)

Reply:
I'll give it a shot though and see what happens